Sources/SATSCore/Components/SectionHeader/SectionHeader.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+public struct SectionHeader<ActionView: View>: View {
+    let title: String
+
+    @ViewBuilder var actionView: ActionView
+    var onClick: (() -> Void)?
+
+    public init(
+        title: String,
+        @ViewBuilder actionView: () -> ActionView = EmptyView.init,
+        onClick: (() -> Void)? = nil
+    ) {
+        self.title = title
+        self.actionView = actionView()
+        self.onClick = onClick
+    }
+
+    public var body: some View {
+        wrapInButtonIfNeeded {
+            HStack {
+                Text(title)
+                    .satsFont(.large, weight: .medium)
+                    .foregroundStyle(Color.onBackgroundPrimary)
+
+                Spacer()
+
+                actionView
+            }
+        }
+        .frame(minHeight: 25)
```

Comment:
maybe better to use one of the spacing values, so the numbers are proportionate to each other